docs/tutorials/authentication/README.md

```diff
@@ -1,210 +1,139 @@
+Create a new `POST` request called "Get Access Token" in the "Authentication Tutorial" collection.
+Set the request URL to `{{TOKEN_ENDPOINT}}`, and the request body to the following:
 
-<img src="./images/set-client-secret.png"/>
-
-#### TOKEN_ENDPOINT
-
-This is the endpoint used to obtain an access token for Machine to Machine connection secured via `CLIENT_ID` and `CLIENT_SECRET`.
-
-If you use Auth0, it will look something like this:
-
-```
-https://TENANT_NAME.us.auth0.com/oauth/token
-```
-
-#### TOKEN_AUDIENCE
-
-Create a new environment variable called `TOKEN_AUDIENCE`.
-
-This value is used to identify the service provider API that the token will be used to access.
-
-You may need to configure your identity provider and token endpoint to support this value.
-
-```
-https://platform.example
-```
-
-#### API_BASE_URL
-
-Create a new environment variable called `API_BASE_URL`.
-
-This value is used to identify the base URL that will be used when making authenticated network requests.
-
-This value should look similar to `TOKEN_AUDIENCE`
-
-```
-https://platform.example/v2
-```
-
-<img src="./images/set-token-endpoint.png"/>
-
-### Setting Up Your Collection
-
-Create a new postman collection called "Traceability Authentication Tutorial".
-
-<img src="./images/create-collection.png"/>
-
-#### Get Your Application Access Token
-
-Create a new postman network request in the collection called "Get Access Token".
-
-```
-POST {{TOKEN_ENDPOINT}}
 ```
-
-Make sure to set the `Accept` header to `application/json`.
-
-Make sure to se the body to:
-
-```json
 {
-  "audience": "{{TOKEN_AUDIENCE}}",
-  "client_id": "{{CLIENT_ID}}",
-  "client_secret": "{{CLIENT_SECRET}}",
-  "grant_type": "client_credentials"
+    "audience": "{{TOKEN_AUDIENCE}}",
+    "client_id": "{{CLIENT_ID}}",
+    "client_secret": "{{CLIENT_SECRET}}",
+    "grant_type": "client_credentials"
 }
 ```
 
-The last step is to add a "Test" which will capture the access token for future use.
-
-```js
-pm.environment.set('access_token', pm.response.json().access_token);
-```
+When you are done, your request should look something like this:
 
-After setting these values, run the request and confirm that your environment now has a new set variable for `access_token`.
+<img src="./resources/get-access-token-request.png"/>
 
-#### Verifying a Credential
-
-Now that you have authenticated as an application, you are ready to make your first authenticated API network request.
-
-Create a new request in your collect called "Verify Credential"
-
-Set the endpoint to be:
-
-```
-POST {{API_BASE_URL}}/credentials/verify
-```
+### Testing
 
-Make sure to set the `Accept` header to `application/json`.
-Make sure to set the `Authorization` to the `{{access_token}}`.
+The "Get Access Token" request does not interact with the system under test, but it is nevertheless valuable to validate the response data so that problems which will cause failures in other requests are called out as early as possible.
```

Comment:
```suggestion
The `Get Access Token` request does not interact with the system under test, but it is nevertheless valuable to validate the response data so that problems which will cause failures in other requests are called out as early as possible.
```